shoop/order_printouts/pdf_export.py

```diff
@@ -16,10 +21,41 @@
     weasyprint = None
 
 
-def render_html_to_pdf(html):
+def _fetch_static_resource_str(resource_file):
+    resource_path = os.path.realpath(os.path.join(settings.STATIC_ROOT, resource_file))
+    if not resource_path.startswith(settings.STATIC_ROOT):
+        raise ValueError(
+            _("Possible file system traversal shenanigan detected with %(path)s") % {"path": resource_file})
+
+    if not os.path.isfile(resource_path):
+        from django.contrib.staticfiles import finders
+        resource_path = finders.find(resource_file)
+
+    if not resource_path:
+        raise ValueError(_("Unable to find: %(path)s") % {"path": resource_file})
+
+    return open(resource_path, "rb").read().decode("UTF-8", "replace")
+
+
+def _custom_url_fetcher(url):
+    if url.startswith("logo:"):
+        thumbnailer = get_thumbnailer(urllib.unquote(url[5:]))
+        thumbnail_options = {"size": (240, 80), "upscale": True}
+        return {"file_obj": thumbnailer.get_thumbnail(thumbnail_options), "mime_type": "image/jpg"}
+    raise ValueError(_("Possible file system traversal shenanigan detected with %(path)s") % {"path": url})
```

Comment:
ValueError message should not be marked for translation (see prev comment for details).